src/style.css

```
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+  
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.big-search {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.big-image {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+  margin-top: 25px;
+}
+
+.small-search {
+  height: 42px;
+  width: 100%;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.small-image {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  margin-left: 13px;
+  margin-top: 15px;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search:hover {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+}
```

```suggestion
}

```
add an empty line to the end of file